Automattic/sensei
# Add question numbering

Part of #3888 

### Changes proposed in this Pull Request

* Add numbering to the question, based on block order inside the quiz block

### Testing instructions

* Edit a lesson, insert a quiz block and add question blocks. Reorder questions.
* Make sure they are numbered correctly
* Make sure they are aligned with the title, and alignment works with 10+ questions


### Screenshot / Video

<img width="624" alt="image" src="https://user-images.githubusercontent.com/176949/106194447-256cf200-61af-11eb-85ac-bb7134327089.png">
